README.md

```diff
@@ -37,18 +84,20 @@ Copy `sample.env` to `.env` and make the following changes within the `.env`:
 
 -   Set `BLOCK_DATA_SOURCE` to either ARCHIVEDB or MINAEXPLORER.
 
-    -   ARCHIVEDB will use the database connection string to get blocks and the current max height, and will expect [hash].json files for the ledgers being processed.
-    -   MINAEXPLORER will use endpoint specified in MINAEXPLORER_GRAPHQL_ENDPOINT (expect: graphql.minaexplorer.com) to get blocks and the staking ledger
+-   ARCHIVEDB will use the database connection string to get blocks and the current max height, and will expect [hash].json files for the ledgers being processed.
+-   MINAEXPLORER will use endpoint specified in MINAEXPLORER_GRAPHQL_ENDPOINT (expect: graphql.minaexplorer.com) to get blocks and the staking ledger
 
 -   Set `COMMISSION_RATE` to the commission your pool charges unknown delegators. (Mina Foundation and O1 delegations have separate configurations in the .env file, and you can set override rates for known delegators -- see "Using payor specific commission rates" below.)
 
 -   Set `MF_COMMISSION_RATE` to the commission your pool charges for Mina Foundation delegations, if any. Mina Foundation maximum rate was .05 from inception until delegation cycle 10, after which the maximum allowable is 0.08.
 
 -   Set `O1_COMMISSION_RATE` to the commission your pool charges for O(1) Labs delegations, if any.
 
+-   Set `INVESTORS_COMMISSION_RATE` to the commission your pool charges for Investors delegations, if any. the defaul value is 0.08.
```

Comment:
I haven't been closely following all discussions but `INVESTORS_COMMISSION_RATE` should be further explained as it's not clear what this is or how it is used. Niggle: typo in default

Reply:
It looks like they need to split MF keys into multiple files - same functionality.